en/api/configuration-generate.md

```diff
@@ -91,9 +91,11 @@ export default {
 }
 ```
 
-The path to the SPA fallback. This file can be used when doing deploys of generated sites to static hosting. It falls back to `mode: 'spa'` when a route isn't generated.
+The path to the fallback HTML file. It should be set as the error page, so that also unknown routes are rendered via Nuxt.
+If set to `true`, it will render as `404.html`.
```

Comment:
To remove all ambiguity i would suggest to explicitly list all the options.

```suggestion
If unset or set to a falsy value, the filename of the fallback HTML file will be `200.html`. If set to `true`, the file will be rendered as `404.html`. In all other cases the filename you provided will be used.
```

Reply:
I've reworded it a little. WDYT? :)